src/templates/static/legal/regulation.jsx

```diff
@@ -150,7 +150,7 @@ const Regulation = () => {
                         />
                     </div>
 
-                    <h3 id='key_information_documents'>{it.L('Key Information Documents')}</h3>
+                    <h3 id='key_information_documents' href='#key_information_documents'>{it.L('Key Information Documents')}</h3>
```

Comment:
why using href? isn't id the way to do it?

Reply:
@nazaninreihani ID does work, and initially it was just with ID and it worked, but if the user is getting redirected from `responsible-trading` for the first time (as in they've never browsed that page again), it doesnt scroll exactly to KID, and after inspecting, I noticed that it scrolls to the beginning of `#accordion`, so in this case for the first time, accordion overrides browser's scroll and scrolls itself, and accordion uses the `href` attribute as selector.

Reply:
Although after thorough testing, i now find that this did not fix the issue, and the issue still persists, but it's seemingly very random. I've tested it a total of 20 times under different configurations (with `href`, without `href`, with `navigation` property on `accordion` init in `regulation.js`, and without) and the issue still only sometimes appears under each of those configurations including combinations of them. Do you have any suggestions for me, @nazaninreihani ?

Reply:
@msamprz have you tried adding id attribute to other matching siblings? 
all h3 of tabs, not just this one

Reply:
@nazaninreihani I tried it now, and out of 15 times, at least 4 times it failed. But still majority of the times it was fine. I may have picked up something that the issue happens when I quickly click on the KID hyperlink (as in not letting the page load fully, or something else).

Reply:
How fast you can click :-D? I removed all contents from responsible trading except the link and clicked on it even before it appeared on screen and it worked

Reply:
when issue happens is url containing the hash?

Reply:
Maybe it helps that my computer is slow 😄, no but it may not be about speed then, I just thought I noticed something like that.

And yes in all circumstances the URL contains the hash. The issue isn't that the scroll or something doesn't work, the issue is that it scrolls to `EU Passport Rights` instead of KID.